pkg/apis/messaging/v1/subscription_validation_test.go

```diff
@@ -484,36 +500,6 @@ func TestSubscriptionImmutable(t *testing.T) {
 			},
 		},
 		want: nil,
-	}, {
-		name: "valid, new Reply",
-		c: &Subscription{
-			Spec: SubscriptionSpec{
-				Channel: getValidChannelRef(),
-				Reply:   getValidReply(),
-			},
-		},
-		og: &Subscription{
-			Spec: SubscriptionSpec{
-				Channel: getValidChannelRef(),
-				Reply:   newReply,
-			},
-		},
-		want: nil,
-	}, {
-		name: "valid, have Reply, remove and replace with Subscriber",
-		c: &Subscription{
-			Spec: SubscriptionSpec{
-				Channel: getValidChannelRef(),
-				Reply:   getValidReply(),
-			},
-		},
-		og: &Subscription{
-			Spec: SubscriptionSpec{
-				Channel:    getValidChannelRef(),
-				Subscriber: getValidDestination(),
-			},
-		},
-		want: nil,
```

Comment:
why are these tests removed?

Reply:
As discussed on Slack, was an issue on my side with understanding the test initially. Anyhow the 2nd test (`valid, have Reply, remove and replace with Subscriber`) does not make sense with the required subscriber IMO anymore, so I removed this.